src/shop-checkout.html

```diff
@@ -120,7 +120,7 @@
           <iron-form id="checkoutForm"
               on-iron-form-response="_didReceiveResponse"
               on-iron-form-presubmit="_willSendRequest">
-            <form method="post" action="/data/sample_success_response.json">
+            <form method="post" action="data/sample_success_response.json">
```

Comment:
Agree, SHOP is not very representative here. Also agree there's a bit of nuance needed in explaining which URLs need to be relative to work in this scheme -- essentially it's URLs to static resources on the primary hosting server. URLs for navigating to "virtual routes" should still be expressed as absolute from the server root, and of course URLs referring to resources on other hosts will remain absolute.